### ISSUE TITLE
Getting notified for all and old feed content once and again

### ISSUE THREAD

**Issue by luckkmaxx** [libre-tube/LibreTube#3880]
### Steps to reproduce

Since 0.15 I get notified on every check with the same videos for all subscriptions. These videos are thoose that fit in the feed timeline (4 weeks?)

### Expected behavior

Notify only once per video

### Actual behavior

Notifies all feed, all the time.

### LibreTube version

0.15

### Android version

13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by luckkmaxx** [comment]
Seems to be fixed reinstalling the app. It was weird because i was notified even if the option was disabled. I think the problem came after upgrading to 0.15. Both in the normal version and the debug.
It is also possible that it was after restoring a backup made minutes before in the same version of the app

**Comment by Bnyro** [comment]
It only seems to do that if there's no new video found since the last time you checked your feed.
Likely just disable notifications for now.
